lisa/secret.py

```diff
@@ -16,6 +16,12 @@
     re.compile(r"^[^.]*?[\\/]?(.)[^\\/]*?(.[.]?[^.]*)$"),
     r"\1***\2",
 )
+# https://xx.core.windows.net/vhds/CentOS.vhd?sp=r&st=xx%2012:10:45&5;CA%aEMpls3D
+# replace as https://xx.core.windows.net/vhds/CentOS.vhd***
+PATTERN_SASURL = (
```

Comment:
The pattern doesn't need to be in the secret lib, it's Azure specified.

Reply:
Have moved this pattern to azure/common.py